utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -712,6 +712,32 @@ int main(int argc, char **argv)
 
                     CHECK(hipHostFree(lutBuffer));
                 }
+                case 35:
+                {
+                    testCaseName = "glitch";
+                    RpptChannelOffsets *rgbOffsets;
+                    CHECK(hipHostMalloc (&rgbOffsets, batchSize * sizeof(RpptChannelOffsets)));
+
+                    for (i = 0; i < batchSize; i++)
+                    {
+                        rgbOffsets[i].r.x = 10;
+                        rgbOffsets[i].r.y = 10;
+                        rgbOffsets[i].g.x = 0;
+                        rgbOffsets[i].g.y = 0;
+                        rgbOffsets[i].b.x = 5;
+                        rgbOffsets[i].b.y = 5;
+                    }
+
+                    startWallTime = omp_get_wtime();
+                    if (inputBitDepth == 0 || inputBitDepth == 1 || inputBitDepth == 2 || inputBitDepth == 5)
+                        rppt_glitch_gpu(d_input, srcDescPtr, d_output, dstDescPtr, rgbOffsets, roiTensorPtrSrc, roiTypeSrc, handle);
+                    else
+                        missingFuncFlag = 1;
+
+                    CHECK(hipHostFree(rgbOffsets));
```

Comment:
Move this outside the loop as mentioned in previous comment

Reply:
Done